charts/hail-search/values.yaml

```diff
@@ -40,7 +40,7 @@ persistentVolume:
   volumeHandle: ''
 
 initContainers: |-
-  {{- range $path, $version := $.Values.global.hail_search.datasetVersions -}}
+  {{- range $path, $version := index $.Values.global "hail-search" "datasetVersions" -}}
```

Comment:
this is [apparently controversial](https://github.com/helm/helm/issues/2192) but seems fine?  `hailSearch` felt a little gross to me.